policy/config/output/lsstComCamDefault.yaml

```diff
@@ -0,0 +1,47 @@
+output:
+  type: LSST_CCD
+  nproc: 8    # Change this to work on multiple CCDs at once.
+  # nfiles: 1   # test
+  nfiles: 9 # Comcam
+
+  header:
+    mjd: *mjd
+    seqnum: *seqnum
+
+  camera: LsstComCam
+
+  exptime: $exptime
+
+  cosmic_ray_rate: 0.0
```

Comment:
what's the unit of this rate? That's where a comment would be very helpful.

Reply:
Added info from https://github.com/LSSTDESC/imSim/blob/56f3f7b46f4e2bddac49193552d614cfa827c923/doc/config.rst#L653